.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: "Publish Cibyl to PyPI"
+on:
+  push:
+    branches:
+      - main
```

Comment:
Do we really want to trigger a release for each PR that goes to main?

Reply:
No, the idea was to impose both conditions, but after double checking the docs it seems that this was wrong. I'll remove the conditional on the branch.